app/views/dashboard/index.html.erb

```diff
@@ -3,7 +3,8 @@
 <p class="lead">OnDemand provides an integrated, single access point for all of your HPC resources.</p>
 <br>
 
-<%= render partial: "motd", locals: { motd: @motd_file  } if @motd_file.exist? %>
+<%# This assumes @motd_file responds to `to_partial_path` %>
+<%= render @motd if @motd %>
```

Comment:
This will always be true. In fact if they do not set the env var `MOTD_PATH`, it will render:

```html
<pre class="motd-monospaced"></pre>
```

This isn't a problem, but was just wondering if this is what you intended.